src/docs/sphinx/user_guide/simple_conduction_tutorial.rst

```diff
@@ -0,0 +1,305 @@
+
+This section is specific to configuration with Lua input files.  If you're just interested in using the C++
+API, you can skip to `Constructing the Mesh`_.
+
+Serac uses Axom's `Inlet <https://axom.readthedocs.io/en/develop/axom/inlet/docs/sphinx/index.html>`_ component
+for defining and extracting information from input files.  Inlet is based on Axom's Sidre component, which provides
+a uniform in-memory layout for simulation data.  We instantiate Inlet with a
+`DataStore <https://axom.readthedocs.io/en/develop/axom/sidre/docs/sphinx/datastore.html>`_ instance (the top-level
+Sidre building block) and the path to the Lua input file, which in this case is ``examples/simple_conduction/conduction.lua``:
+
+.. literalinclude:: ../../../../examples/simple_conduction/with_input_file.cpp
+   :start-after: _inlet_init_start
+   :end-before: _inlet_init_end
+   :language: C++
+
+We then define the schema for the input file.  Instead of defining the structure of the input file in one place, Inlet
+allows Serac to separate its schema definition logic into functions that are responsible for defining just one component
+of the schema.  Since our input file contains information required for mesh construction, for Serac's ``ThermalConduction`` module,
+and for the selected output type, we use Inlet to define the corresponding schemas:
+
+.. literalinclude:: ../../../../examples/simple_conduction/with_input_file.cpp
+   :start-after: _inlet_schema_start
+   :end-before: _inlet_schema_end
+   :language: C++
+
+.. note::
+  Since Serac's schema definition functions are independent of their location in the input file, we add a struct whose name
+  corresponds to the location in ``conduction.lua`` and pass that to the appropriate schema definition function.
+
+Because input file parsing happens when the schema is defined, there is no need to call a separate ``parse`` function.
+We conclude Inlet's setup by calling the ``verify`` method, which ensures that all required data is present and meets
+any other constraints:
+
+.. literalinclude:: ../../../../examples/simple_conduction/with_input_file.cpp
+   :start-after: _inlet_verify_start
+   :end-before: _inlet_verify_end
+   :language: C++
+
+.. hint::
+  The ``SLIC_ERROR_ROOT_IF`` macro is part of Serac's :ref:`logging-label` functionality,
+  which is enabled as part of the ``serac::initialize`` function described above.
+
+Constructing the Mesh
+---------------------
+
+In this introductory example, we will use a simple square mesh with 10 quadrilateral elements in each space dimension
+for 100 elements total.  Once created, the primary mesh must always be registered with the ``StateManager``: 
+
+Using C++
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/without_input_file.cpp
+   :start-after: _create_mesh_start
+   :end-before: _create_mesh_end
+   :language: C++
+
+After constructing the serial mesh, we call ``refineAndDistribute`` to distribute it into a parallel mesh.
+
+Using Lua
```

Comment:
These jump out a lot.  Can you try making them bold instead of a heading?

**Using Lua:** ?